packages/js-moi-providers/types/jsonrpc.d.ts

```diff
@@ -337,4 +339,39 @@ export interface LogFilter {
     address: string;
     height: [start: number, end: number];
     topics?: NestedArray<string>;
-}
\ No newline at end of file
+}
+
+
+export interface Log {
+    address: string;
+    topics: string[];
+    data: string;
+    ix_hash: string;
+    ts_hash: string;
+    participants: Participants;
+}
+
+export interface LogFilter {
+    address: string;
+    height: [start: number, end: number];
+    topics?: NestedArray<string>;
+}
+export type NewTesseractsByAccount = {
+    event: WebSocketEvent.NewTesseractsByAccount;
+    params: { address: string };
+};
+
+export type NewLogs = { event: WebSocketEvent.NewLog; params: LogFilter };
+
+export type ProviderEvents =
+    | keyof WebsocketEventMap
+    | { event: string; params: any };
+
+export interface WebsocketEventMap {
+    [WebSocketEvent.NewTesseracts]: [tesseract: Tesseract];
+    [WebSocketEvent.NewPendingInteractions]: [interactionHash: string];
+    [WebSocketEvent.Connect]: [];
+    [WebSocketEvent.Error]: [error: unknown];
+    [WebSocketEvent.Close]: [];
+    [WebSocketEvent.Reconnect]: [attempt: number];
+}
```

Comment:
has to be move to websocket.d.ts

Reply:
https://github.com/sarvalabs/js-moi-sdk/pull/91/commits/981b0c690a186023f9d24ccadf231a297a4e76af
https://github.com/sarvalabs/js-moi-sdk/pull/91/commits/4d43d072fb79aef9e75c5e2ed4739205da987596